### DISCUSSION TITLE
Disable spinning circle animation when q-expansion-item is clicked

### DISCUSSION THREAD

**Discussion by Kellrkind** [quasarframework/quasar#11258] [General - Components / Directives / etc] [answered]
Hey guys,

I'm new to quasar and startet building a little project.
On one page I've implemented a q-list with q-expansion-item (repeated through a v-for).

Everytime when I'm running the application and try to click onto an entry an animation appears.
It's a circle getting larger (as you can see in the screenshot attached).

It's very annoying and i just want to turn it off, but haven't found a way to do it.
Can you help me?

![2021-11-08_22h29_27](https://user-images.githubusercontent.com/7069944/140822331-814935bb-dd72-42d7-84bf-901985785eeb.png)

**Comment by MilosPaunovic** [comment]
Please post a Codepen reproduction of the issue or at least the code you've written for the `QExpansionItem`.

**Comment by Kellrkind** [comment]
Here is my code for the template section :)

```
<template>
  <q-page class="column">

    <q-list class="rounded-borders bg-white q-mt-sm">

      <q-item>
        <q-item-section>
          <q-item-label class="text-h5 text-color" style="text-align: center">News über unsere NZ</q-item-label>
        </q-item-section>
      </q-item>

      <q-expansion-item
        v-for="news in newsList"
        :key="news.title"
        :label="news.title"
        :caption="formatDate(news.date)"
        v-ripple
        expand-separator
        expand-icon-class="positive"
        v-model="news.opened"
        :header-class="{
          'header-blue':news.opened
        }"
        :content-inset-level="1"
        popup>

        <q-separator/>

        <q-card>
          <q-card-section> {{ news.fullText }}</q-card-section>
        </q-card>

      </q-expansion-item>

    </q-list>

  </q-page>
</template>
```

**Comment by MilosPaunovic** [comment, marked as answer]
Just remove the `v-ripple` directive from your `QExpansionItem`.

**Comment by Kellrkind** [comment]
Wait that's all?
In all tutorials I've seen this code snippet is used.

Thx mate! :)
Just on follow-up question. What does `v-ripple` exactly do?

**Comment by MilosPaunovic** [comment]
Yup, that's it!
You can read more about the `v-ripple` [here](https://quasar.dev/vue-directives/material-ripple).

**Comment by pdanpdan** [comment]
https://quasar.dev/vue-directives/material-ripple#usage

If you really need a ripple on header it's a little more complicated:
https://codepen.io/pdanpdan/pen/JjyvwwE?editors=1000
